### PR TITLE
Added additional information to Coupon Codes page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to Coupon Codes page

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/price-rules-cart-coupon.html

### CODE REVIEW

File: src/marketing/price-rules-cart-coupon.md [hunk 1]
```diff
@@ -106,11 +106,24 @@ The generation of discount coupons is an asynchronous operation, which executes
...
    _Generated Codes_
 
+|Field|Description|
+|--- |--- |
+|Coupon Code|A unique code of coupon that was created and can be used for receiving special conditions.|
+|Created|The date when the coupon code was created.|
+|Used|Indicates if the coupon was used.|
+|Times Used|Indicates how many times the coupon code was used.|
+
+Coupon codes can be exported to CSV or Excel XML files. For doing that select the needed file format and click <span class="btn">Export</span>.
```
Comment (leslietilling) [inline src/marketing/price-rules-cart-coupon.md:123]:
```suggestion
You can export coupon codes to a CSV or Excel XML file by selecting the file format and clicking <span class="btn">Export</span>.
```

File: src/marketing/price-rules-cart-coupon.md [hunk 2]
```diff
@@ -106,11 +106,24 @@ The generation of discount coupons is an asynchronous operation, which executes
...
+
+To delete coupon codes, select needed codes from the list, select `Delete` from the **Actions** drop-down and click <span class="btn">Submit</span>.
```
Comment (leslietilling) [inline src/marketing/price-rules-cart-coupon.md:125]:
```suggestion
To delete coupon codes, select one or more codes from the list, select `Delete` from the **Actions**  selector, and click <span class="btn">Submit</span>.
```

File: src/marketing/price-rules-cart-coupon.md [hunk 3]
```diff
@@ -106,11 +106,24 @@ The generation of discount coupons is an asynchronous operation, which executes
 
 1. When complete, click <span class="btn">Generate</span>.
 
-   The list of generated codes appears below.
+   `Message is added to queue, wait to get your coupons soon` message is appeared. Make sure that cron is running on your server.
```
Comment (leslietilling) [inline src/marketing/price-rules-cart-coupon.md:109]:
```suggestion
   The system displays `Message is added to queue, wait to get your coupons soon`. Make sure that cron is running on your server.
```

File: src/marketing/price-rules-cart-coupon.md [hunk 4]
```diff
@@ -106,11 +106,24 @@ The generation of discount coupons is an asynchronous operation, which executes
...
+
+   After the cron job was processed, the list of generated codes appears below.
```
Comment (leslietilling) [inline src/marketing/price-rules-cart-coupon.md:111]:
```suggestion
   After the cron job was processed, the list of generated codes appears.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by m2-github-services** [comment]
@leslietilling the branch with the code is successfully imported into the `magento-devdocs/merchdocs` repository. Branch name: `imported-DmitryTsymbal-merchdocs-1700`.

**Pull request merged**